src/eduid_userdb/orcid.py

```diff
@@ -54,7 +70,10 @@ def __init__(
                 created_by=application,
                 created_ts=created_ts,
             )
-        Element.__init__(self, data)
+        elif 'created_ts' not in data:
+            data['created_ts'] = True
+
+        Element.__init__(self, data, called_directly=called_directly)
```

Comment:
super()

Reply:
This one?

Reply:
In this case we need it like this, since the superclass of `OidcToken`is not `Element` but `OidcElement`, that raises `NotImplementedError` in its `__init__` - which is there to introduce the `raise_on_unknown` param so that we can use `OidcElement.from_dict` for both `OidcToken` and `OidcAuthorization`

Reply:
that warrants a comment then

Reply:
Rather than raising, I might have called `super.__init__` w/o the `raise_on_unknown` param - but then we would have an unused param, so I think this is more correct?

Reply:
> that warrants a comment then

Yep you are right